odoo/addons/base/models/res_partner.py

```diff
@@ -151,7 +155,7 @@ def default_get(self, default_fields):
     parent_name = fields.Char(related='parent_id.name', readonly=True, string='Parent name')
     child_ids = fields.One2many('res.partner', 'parent_id', string='Contact', domain=[('active', '=', True)])  # force "active_test" domain to bypass _search() override
     ref = fields.Char(string='Reference', index=True)
-    lang = fields.Selection(_lang_get, string='Language', default=lambda self: self.env.lang,
+    lang = fields.Selection(_lang_get, string='Language',
```

Comment:
Why not a computed editable stored field :D ?

Reply:
@tde-banana-odoo : That what I thought when I read your onchange to compute PR :D,
I could then delete completely the default_get, while I'm at it, correct ?

Reply:
I did that as a workaround downstream: https://github.com/OCA/partner-contact/pull/935/commits/67b597f3011f4ee780199f9bbb0bc7f34dc23196